### DISCUSSION TITLE
Windows 10

### DISCUSSION THREAD

**Discussion by darrenmarcel810705** [Fs00/Win10BloatRemover#8] [General]
Requesting change active hours to be removed/deleted.

**Comment by Fs00** [comment]
I'm not sure on what you mean.
Are you referring to the [suggestion to change active hours in Windows Update settings page](https://www.kunal-chowdhury.com/2020/06/windows-10-recommends-active-hours.html?m=1)?

**Comment by darrenmarcel810705** [comment]
Disabling the change active hours

**Reply by Fs00** [reply]
This is exactly what isn't clear to me.
You'd like to have the tool hide/disable the option to change active hours in Settings app?

**Comment by darrenmarcel810705** [comment]
Yeah disable it for full control of windows.

**Reply by Fs00** [reply]
My personal take on this is that hiding/disabling an option, therefore becoming unable to change it, reduces your control on the system. And disabling automatic Windows updates already makes active hours ineffective, so I really can't understand what you'd like to achieve here 🤔
